src/style.css

```diff
@@ -1,3 +1,86 @@
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo_image {
+  width: 40px;
+  height: 40px;
+}
+
+.navigation {
+  align-items: center;
+  display: flex;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding-left: 0;
+}
+
+.nav-item {
+  text-align: center;
+  display: flex;
+}
+
+/* .nav-item:first-child {
+  margin-left: 0;
+}
+
+.nav-item:last-child {
+  margin-left: 0;
+  padding-right: 50px;
+} */
+
+.nav_link {
+  position: relative;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: Roboto, Arial, sans-serif;` should have a generic font family as a fallback.